internal/bminventory/inventory.go

```diff
@@ -778,11 +778,11 @@ func (b *bareMetalInventory) refreshClusterHosts(ctx context.Context, cluster *c
 		if err = tx.Take(&host, "id = ? and cluster_id = ?",
 			h.ID.String(), cluster.ID.String()).Error; err != nil {
 			log.WithError(err).Errorf("failed to find host <%s> in cluster <%s>",
-				h.ID.String(), cluster.ID.String())
+				common.GetHostnameForMsg(h), cluster.ID.String())
```

Comment:
Do we want hostname in the logs as well? not sure if it's a good idea

Reply:
IMO it is clearer.
What do you think? @ronniel1 @avishayt @eranco74

Reply:
When trying to debug i think that it will be easier to use host-id

Reply:
Logs should have ID since it doesn't change

Reply:
I changed only the event message